### PR TITLE
Update chiseled content

### PR DESCRIPTION
**Pull request by richlander**
- Updated chiseled content
- Moved content to better places

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/whats-new/dotnet-8/containers.md](https://github.com/dotnet/docs/blob/818bef715da5f341963f99a997bc4d7f64d735c8/docs/core/whats-new/dotnet-8/containers.md) | [What's new in containers for .NET 8](https://review.learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-8/containers?branch=pr-en-us-40267) |

### CODE REVIEW

File: docs/core/whats-new/dotnet-8/containers.md [hunk 1]
```diff
@@ -77,14 +75,21 @@ As part of an effort to improve containerization performance, new ASP.NET Docker
 
 There is a caveat to be aware of. Since composites have multiple assemblies embedded into one, they have tighter version coupling. Apps can't use custom versions of framework or ASP.NET binaries.
 
-Composite images are available for the Alpine Linux, Jammy Chiseled, and Mariner Distroless platforms from the `mcr.microsoft.com/dotnet/nightly/aspnet` repo. The tags are listed with the `-composite` suffix on the [ASP.NET Docker page](https://hub.docker.com/_/microsoft-dotnet-nightly-aspnet).
+Composite images are available for the Alpine Linux, Ubuntu ("jammy") Chiseled, and Mariner Distroless platforms from the `mcr.microsoft.com/dotnet/nightly/aspnet` repo. The tags are listed with the `-composite` suffix on the [ASP.NET Docker page](https://hub.docker.com/_/microsoft-dotnet-nightly-aspnet).
```
Comment (richlander) [inline docs/core/whats-new/dotnet-8/containers.md:78]:
```suggestion
Composite images are available for the Alpine Linux, Ubuntu ("jammy") Chiseled, and Mariner Distroless platforms from the `mcr.microsoft.com/dotnet/aspnet` repo. The tags are listed with the `-composite` suffix on the [ASP.NET Docker page](https://hub.docker.com/_/microsoft-dotnet-aspnet).
```

File: docs/core/whats-new/dotnet-8/containers.md [hunk 2]
```diff
@@ -16,24 +16,12 @@ This article describes new features in containers for .NET 8.
 
 The following changes have been made to .NET container images for .NET 8:
 
-- [Generated-image defaults](#generated-image-defaults)
-- [Debian 12](#debian-12)
 - [Non-root user](#non-root-user)
+- [Debian 12](#debian-12)
 - [Chiseled Ubuntu images](#chiseled-ubuntu-images)
 - [Build multi-platform container images](#build-multi-platform-container-images)
 - [ASP.NET composite images](#aspnet-composite-images)
```
Comment (richlander) [inline docs/core/whats-new/dotnet-8/containers.md:24]:
```suggestion
```

**Review by gewarren — APPROVED** [review]
Thanks!

**Comment by richlander** [comment]
I'll wait on merging this until Monday to give the team a chance to review it (or if I find more stuff to update).

**Review by mthalman — APPROVED** [review]

File: docs/core/whats-new/dotnet-8/containers.md [hunk 3]
```diff
@@ -77,14 +74,21 @@ As part of an effort to improve containerization performance, new ASP.NET Docker
...
+- [Generated-image defaults](#generated-image-defaults)
 - [Performance and compatibility](#performance-and-compatibility)
 - [Authentication](#authentication)
 - [Publish to tar.gz archive](#publish-to-targz-archive)
 
+### Generated-image defaults
+
+`dotnet publish` can produce container images. It defaults to producing [`non-root` images](#non-root-user), which helps your apps stay secure-by-default. Change this default at any time by setting the `ContainerUser` propery, for example with `root`.
+
+The default container tag is now `latest`. This default is in line with other tooling in the containers space and makes containers easier to use in inner development loops.
```
Comment (mthalman) [inline docs/core/whats-new/dotnet-8/containers.md:90]:
```suggestion
The default output container tag is now `latest`. This default is in line with other tooling in the containers space and makes containers easier to use in inner development loops.
```

**Review by mthalman — APPROVED** [review]

**Pull request merged**